web-application/webpage-with-ajax-call/index.html.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>OMDB API Ajax Request</title>
+<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+
+<div id="container">
+    <h2>OMDB API Ajax Request</h2>
+    <button id="ajaxButton"  class="butt">Make Ajax Request</button>

```

Comment:
Don't use offensive words in the code

Reply:
Apologies for the oversight. Cleared that inside the code.